testing/test_aux.out

```diff
@@ -41,7 +41,7 @@ RESULT pass security.firmware version found: ?\xFF\xFF\x19,>u\x08\x00no
 dhcp requests 2 1
 01: ['01:brute:1', '01:macoui:1']
 02: ['02:nmap:1', '02:ping:1', '02:tls:1']
-03: ['03:nmap:1', '03:udmi:1']
+03: ['03:nmap:1']
```

Comment:
I believe this is a side effect of DAQ gracefully skipping the udmi tests if there isn't a GCP_SERVICE_ACCOUNT environment variable present/a proper gcp config in `system.conf`. This makes the `aux` Travis fail on PRs (like this one) since `cloud.udmi.pointset` is skipped on port 3, rather than producing a fail as expected. I can look into this, but it's probably for another PR?